README.md

```diff
@@ -95,23 +97,16 @@ Para executar os seeds (arquivos responsáveis pela inserção de todos os dados
 yarn sequelize db:seed:all
 ```
 
-Uma vez que os arquivos de migrations já foram executados e é necessária alguma atualização no schema de determinada(s) tabela(s) é necessário re-executar as migrations com o comandos:
+### Comandos alternativos
```

Comment:
O comando acima gerou esse erro:

yarn run v1.22.4
$ /home/ubuntu/pdc-deploy/node_modules/.bin/sequelize db:seed:all

Sequelize CLI [Node: 12.18.3, CLI: 5.5.1, ORM: 5.21.12]

Loaded configuration file "src/config/database.js".
== 20200618182333-add-generos: migrating =======

ERROR: relation "Genero" does not exist

error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.

Reply:
Creio que o Sr. executou o **yarn sequelize db:seed:all** antes do **yarn sequelize db:migrate**, por isso está dizendo que a tabela Genero não existe.